pkg/proxy/ipvs/proxier_test.go

```diff
@@ -131,7 +131,8 @@ func NewFakeProxier(ipt utiliptables.Interface, ipvs utilipvs.Interface, ipset u
 	// initialize ipsetList with all sets we needed
 	ipsetList := make(map[string]*IPSet)
 	for _, is := range ipsetInfo {
-		ipsetList[is.name] = NewIPSet(ipset, is.name, is.setType, false, is.comment)
+		set, _ := NewIPSet(ipset, is.name, is.setType, false, is.comment)
+		ipsetList[is.name] = set
 	}
```

Comment:
Hey, you've clearly done a lot of hard work on this. Do you have any idea on why the errors are still coming? Thanks!

Reply:
@Aut0R3V Hi, previously, we need to new `pkg/util/ipset/ipset.go` IPSet and call validate() manually, validate may return error, for the current time, The `pkg/util/ipset/ipset.go` NewIPSet() will call validate() automatically, so there will be an error.  Previously `validate()` is not called when `pkg/proxy/ipvs/ipset.go` NewIPSet() new the `pkg/util/ipset/ipset.go` IPSet.